acceptance-tests.md

```diff
@@ -46,18 +53,24 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
         6. The user will be able to authenticate using Keycloak and their institutional login.
+    * Acceptance tests:
+        1. For a given project named `myproject` for example, [Click here to check that the given project is listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=project.openshift.io%7Ev1%7EProject&name=myproject).
 2. **Deactivate a project or resource allocation**
     * _As an administrator, I should be able to archive any project or resource allocation and release the resources associated with it back to the pool._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. A Coldfront admin can navigate to the project
             1. They can archive a project and expire all associated allocations by clicking ‘archive project’ by navigating to the project and clicking ‘archive project’.
             2. They can navigate to an allocation, set the status to “Denied”, and update the allocation
         2. Disabling an allocation will delete the associated OpenShift namespace, which differs from OpenStack behavior which simply disables the project.
+    * Acceptance tests:
+        1. For a given project named `myproject` for example, [Click here to check that the given project is no longer listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=project.openshift.io%7Ev1%7EProject&name=myproject).
 3. **Manage a project as a PI.**
     * _As a PI, I should be able to manage and share my project with others on the team, but no one except the PI and the administrator should be able to remove the project._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. A PI can add keycloak users to a ColdFront project under the ‘users’ section in the given project ([https://nerc-project.github.io/nerc-docs/get-started/get-an-allocation/#adding-and-removing-user-from-the-project](https://nerc-project.github.io/nerc-docs/get-started/get-an-allocation/#adding-and-removing-user-from-the-project))
         2. From here, a PI can set the user to a particular role. It seems the ‘manager’ role is the one that lets users create and remove allocations by delegating PI role/responsibilities. Some other roles should be set for other general users who are only going to use the allocated NERC resources.
+    * Acceptance tests:
+        1. For a given project named `myproject`, and a given user named `myuser`, and a given role named `manager` for example, [Click here to check that the given project contains a RoleBinding with a "Role ref" of `manager`, a "Subject kind" of `User`, and a "Subject name" of `myuser` in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/search/ns/myproject?kind=rbac.authorization.k8s.io%7Ev1%7ERoleBinding&rowFilter-role-binding-kind=namespace).
 
 
```

Comment:
Use case 2.1 about a user being able to add new namespaces is not covered here.  Is it covered in the OpenShift basic tests?

Reply:
A non-admin user cannot request a new namespace.  The steps for a PI to request the namespace are documented now, so this is resolved for the acceptance tests.  The non-admin namespace request function needs to be removed from the use case document.